RecoEgamma/EgammaIsolationAlgos/plugins/isolation_cones/PhotonPFIsolationWithMapBasedVeto.cc

```diff
@@ -132,9 +132,9 @@ bool PhotonPFIsolationWithMapBasedVeto::isInIsolationCone(const reco::CandidateP
   // dealing here with patObjects: miniAOD case
   if ( aspacked.get() )    
   {
-    inFootprint = isInFootprint(aspat_photonptr ->associatedPackedPFCandidates(),aspacked);
-    
-    //checking if the charged candidates come from the appropriate vertex
+    //inFootprint = isInFootprint(aspat_photonptr ->associatedPackedPFCandidates(),aspacked);
```

Comment:
as far as I understand that means that this should be introduced in 
DataFormats/EgammaCandidates/interface/Photon.h
or you mean something else?